app/gui/src/project-view/components/MarkdownEditor/DocumentationImage.vue

```diff
@@ -34,16 +34,55 @@ const title = computed(() =>
 
 const alt = props.alt ? props.alt : DEFAULT_ALT_TEXT
 
+const isYouTubeVideo = computed(() => {
+  return props.src.match(/https:\/\/www\.youtube\.com\/embed\/[^\/]+/)
+})
+
 onUnmounted(() => {
   if (data.value?.ok) data.value.value.dispose?.()
 })
 </script>
 
 <template>
   <img
+    v-if="!isYouTubeVideo"
     :src="data?.ok ? data.value.url : ''"
     :alt="alt"
     :title="title"
     :class="{ uploading: data?.ok && data.value.uploading?.value }"
   />
+  <div v-if="isYouTubeVideo" class="youtube-video-container">
+    <div>
+      <iframe
+        :src="data?.ok ? data.value.url : ''"
+        :title="title"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        referrerpolicy="strict-origin-when-cross-origin"
+        allowfullscreen>
+      </iframe>
+    </div>
+  </div>
```

Comment:
I would put youtube case first, and image for `v-else`

```suggestion
  <div v-if="isYouTubeVideo" class="youtube-video-container">
    <div>
      <iframe
        :src="data?.ok ? data.value.url : ''"
        :title="title"
        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
        referrerpolicy="strict-origin-when-cross-origin"
        allowfullscreen>
      </iframe>
    </div>
  </div>
  <img
    v-else="!isYouTubeVideo"
    :src="data?.ok ? data.value.url : ''"
    :alt="alt"
    :title="title"
    :class="{ uploading: data?.ok && data.value.uploading?.value }"
  />
```

Reply:
`v-else`, not `v-else="!isYouTubeVideo"`, right?

Reply:
Rigtht, edited original comment.